master/static/srv/get_monthly_aggregated_allocation.sql

```diff
@@ -1,25 +1,63 @@
 WITH const AS (
     SELECT
         daterange($1 :: date, $2 :: date, '[]') AS period
+),
+months AS (
```

Comment:
after reading a few times, with some parameters, seems this query can produce the other. i don't think it's a big enough deal to change though.